core/src/functions/content.functions.ts

```diff
@@ -147,9 +148,11 @@ export const generateBibleDocument = (
                     activeParagraph = (<IBibleContentGeneratorGroup<'paragraph'>>_group).meta;
                 } else if (_group.groupType === 'indent') {
                     // => this group has a level (numeric) modifier
+                    if (activeModifiers['indentLevel'] !== 2) {
```

Comment:
as mentioned in slack, this is probably to work around a bug with indents? Hopefully passing a cloned state to a new BiblePhrase fixes this? otherwise could you set me up to reproduce the bug then i look into it.